Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM python:3.11
 ARG ANSIBLE_VERSION=stable-2.16
 RUN pip install --no-cache-dir --disable-pip-version-check https://github.com/ansible/ansible/archive/${ANSIBLE_VERSION}.tar.gz
-COPY ./requirements.txt /tmp/requirements.txt
-RUN pip install --no-cache-dir --disable-pip-version-check --requirement /tmp/requirements.txt
+COPY ./requirements.txt ./requirements-dev.txt /tmp/
+RUN pip install --no-cache-dir --disable-pip-version-check --requirement /tmp/requirements.txt && pip install --no-cache-dir --disable-pip-version-check --requirement /tmp/requirements-dev.txt
```

Comment:
```suggestion
RUN pip install --no-cache-dir --disable-pip-version-check --requirement /tmp/requirements*.txt
```

Reply:
@glimchb I rectified it , I added the wildcard